docs/standardisation-scope.adoc

```diff
@@ -0,0 +1,199 @@
+= Standardisation scope of Cypher
+
+One of the challenges faced by this effort to standardise Cypher has been to choose the set of language features to standardise.
```

Comment:
"One of the challenges faced by this effort to standardise Cypher has been to choose the set of language features to standardise." -->
"One of the challenges faced by the standardisation of Cypher has been to choose which language features to include."